cinn/frontend/op_mapper_registry.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <variant>
+
+#include "cinn/common/macros.h"
+#include "cinn/common/target.h"
+#include "cinn/frontend/net_builder.h"
+#include "cinn/frontend/paddle/cpp/op_desc.h"
+#include "cinn/frontend/syntax.h"
+#include "cinn/hlir/framework/scope.h"
+#include "cinn/utils/registry.h"
+
+namespace cinn {
+namespace frontend {
+
+class OpMapperContext {
+ public:
+  OpMapperContext(hlir::framework::Scope* scope,
+                  const common::Target& target,
+                  NetBuilder* builder,
+                  absl::flat_hash_map<std::string, Variable>* var_map,
+                  absl::flat_hash_map<std::string, std::string>* var_model_to_program_map)
+      : scope_(scope),
+        target_(target),
+        builder_(builder),
+        var_map_(var_map),
+        var_model_to_program_map_(var_model_to_program_map) {}
+
+  hlir::framework::Scope* scope_{nullptr};
+  const common::Target& target_;
+  NetBuilder* builder_{nullptr};
+  absl::flat_hash_map<std::string, Variable>* var_map_{nullptr};
+  // map from var in Paddle model to var name in program.
+  absl::flat_hash_map<std::string, std::string>* var_model_to_program_map_{nullptr};
```

Comment:
类的属性还是应该声明为private的，或者你这里类比较简单，可以声明为struct。
```suggestion
struct OpMapperContext {
  OpMapperContext() {
    // ...
  }
  NetBuilder* builder{nullptr};
};
```

Reply:
好的，我再提个PR修改下